example/metro.config.js

```diff
@@ -1,19 +1,33 @@
-// https://github.com/facebook/react-native/issues/24065#issuecomment-537489786
+const exclusionList = require('metro-config/src/defaults/exclusionList');
 
```

Comment:
Did this new `metro.config.js` come from somewhere?  If so, would be help to add a comment with a link to it.

Reply:
could ref: https://github.com/callstack/react-native-builder-bob/blob/1000cd7c4d99647f4ad2e90cfcc5874697e270aa/packages/create-react-native-library/templates/common/example/metro.config.js